### ISSUE TITLE
Application output: 2017/06/15 01:32:53 exit status 1

### ISSUE THREAD

**Issue by barkdoll** [FooSoft/yomichan-import#8]
The local server port doesn't seem to start in order to import the local dictionary. It shows me a time stamp followed by 'exit status 1' as if I already stopped it on the localhost.

I'm using Windows 10. I have tried changing the port number a few times. Nothing changed. Do you know how I could get the server port running?

**Comment by barkdoll** [comment]
Hey for what it's worth, I found a workaround. I had the CATALOGS file along with the dictionary's other EPWING files in a folder named '大辞林 第２版'.

I think it may be because either there was a space in the directory name or because it had illegal characters in the folder name (Japanese characters don't comply with ASCII or something like that?). 

What I did to fix this was:

1. Copy folder and paste to another location on PC.
2. Rename the folder taking out space and Japanese characters (renamed **大辞林 第２版** to **_daijirin_**)
3. Browse to new folder's path and select the CATALOGS file from it.
4. Import dictionary...

Just a quick fix in case anyone else has an issue with getting it to process the file and send to the server host.

P.S. - Awesome work by the way. I love Yomichan. I want to eventually develop apps to help the Japanese/language-learning community as well. You are an inspiration!
